functional/kotlin/build.gradle

```diff
@@ -0,0 +1,16 @@
+plugins {
+    id 'org.jetbrains.kotlin.jvm'
+    id 'org.jmailen.kotlinter'
+    id 'java-gradle-plugin'
+}
+
+repositories {
+    jcenter()
+    google()
+}
+
+dependencies {
+    runtimeOnly project(':')
+    runtimeOnly 'org.jetbrains.kotlin:kotlin-gradle-plugin'
```

Comment:
I verified that this setup would catch the mistake I did in https://github.com/jeremymailen/kotlinter-gradle/pull/124#discussion_r368314350